src/software/geom/util.h

```diff
@@ -160,13 +161,13 @@ bool collinear(const Point &a, const Point &b, const Point &c);
  *
  * @param radius the radii of the obstacles.
  *
- * @return The best point to aim for in the target area and the size of the open interval
- *         through which a shot from the current point to the best point would go. If no
- *         shot could be found, returns std::nullopt
+ * @return The Shot with the best point to aim for in the target area and the size of the
+ * open interval through which a shot from the current point to the best point would go.
+ * If no shot could be found, returns std::nullopt
```

Comment:
Again, no need to specify what a shot contains, just saying it returns a shot is sufficient (see above comments for examples).